### PR TITLE
eat webpack-dev-server logs

### PR DESCRIPTION
**Pull request by fivethreeo**

### CODE REVIEW

**Comment by nimaa77** [comment]
is this ready?

**Comment by fivethreeo** [comment]
Unless you see any issues with it. Yes :)

**Comment by nimaa77** [comment]
nice job 👍 
my suggestion is to remove `> Webpack dev server started on port 3001` message or (show it in gray color)

feel free to merge it ...

File: examples/basic/src/index.js [hunk 1]
```diff
@@ -2,16 +2,24 @@ import express from 'express';
...
+try {
+  green = require('chalk').green;
+} catch (error) {
+  green = (msg) => msg;
+}
+
 if (module.hot) {
   module.hot.accept('./server', function() {
-    console.log('🔁  HMR Reloading `./server`...');
+    console.log(green('🔁 HMR Reloading `./server`...'));
```
Comment (nimaa77) [inline examples/basic/src/index.js]: this is not necessary but if you think it's good to have it in green color, we have to update all the examples ...

File: packages/razzle/config/razzleDevServer.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  constructor(compiler, options = {}, _log) {
+    const verbose = options.verbose || false;
+    delete options['verbose'];
+    super(compiler, options, _log);
+    this.verbose = verbose;
+  }
+
+  showStatus() {
+
+    console.log(chalk.green(`> Webpack dev server started on port ${this.options.port}`));
```
Comment (nimaa77) [inline packages/razzle/config/razzleDevServer.js]: maybe remove this line or if you prefer to don't remove it just keep it

but put an if statement that doesn't show this message when we start this with `--verbose` ...

**Review by nimaa77 — APPROVED** [review]

**Comment by fivethreeo** [comment]
Valid points, changing :)

**Review by nimaa77 — APPROVED** [review]

**Comment by nimaa77** [comment]
GG 🥇

File: packages/razzle/scripts/start.js [hunk 3]
```diff
@@ -131,7 +135,7 @@ function compile(config) {
   try {
     compiler = webpack(config);
   } catch (e) {
-    printErrors('Failed to compile.', [e]);
+    printErrors('Failed to compile.', [e], verbose);
```
Comment (karlnorling) [inline packages/razzle/scripts/start.js:138]: Verbose isn't in scope here.

```
(node:98090) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'verbose' of undefined
    at compile (/Users/karl/code/karlnorling.github.io/node_modules/razzle/scripts/start.js:152:59)
    at /Users/karl/code/karlnorling.github.io/node_modules/razzle/scripts/start.js:67:34
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
```

Should either be `process.env.VERBOSE` or come from somewhere inside the `config` object passed in t the `compile` function.

Comment (fivethreeo) [reply packages/razzle/scripts/start.js:138]: Fixed in canary branch now :)

**Pull request merged**